AdobeStockImage/Model/AssetIndexer.php

```diff
@@ -10,8 +10,6 @@
 use Magento\AdobeStockAssetApi\Api\AssetRepositoryInterface;
 use Magento\Framework\Api\SearchCriteriaBuilder;
 use Magento\Framework\App\Filesystem\DirectoryList;
-use Magento\Framework\App\ResourceConnection;
-use Magento\Framework\DB\Adapter\AdapterInterface;
 use Magento\Framework\Filesystem;
 use Magento\Framework\Filesystem\Directory\ReadInterface;
 use Magento\Framework\Filesystem\Driver\File;
```

Comment:
Like an alternative, you can avoid using direct class loading by using the `use Magento\Framework\FileSystem\DriverInterface`. To inject it you can declare a new constructor parameter and set the `Magento\Framework\Filesystem\Driver\File` through the di.xml. See this as an example: https://github.com/magento/adobe-stock-integration/pull/940/files#diff-adb30514bd65a37c94ac4bcf1fccb969R43

Reply:
Thank you !